ktor-server/ktor-server-plugins/ktor-server-graphql/jvm/src/io/ktor/server/plugins/graphql/GraphQL.kt

```
@@ -0,0 +1,213 @@
+    val graphql = config.graphQLFactory?.invoke(schema)
+        ?: GraphQL.newGraphQL(schema).build()
+
+    val contextFactory = config.contextFactory
+
+    val json = Json { ignoreUnknownKeys = true }
+
+    val route = route(config.endpoint) {
+        post {
+            val request = try {
+                json.decodeFromString<GraphQLRequest>(call.receiveText())
+            } catch (cause: SerializationException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)